_pages/saml.md

```diff
@@ -66,15 +66,68 @@ Here are values needed to configure your service provider (SP) to work with logi
   ```
 
 - **x509 Public Certificate**
-  The public certificate is used to validate the authenticity of SAML requests received from login.gov, a minimum of 2048 bits. We publish this public certificate from in our [Metadata endpoint](#metadata).
+  The public certificate is used to validate the authenticity of SAML requests received from login.gov, a minimum of 2048 bits. We publish this public certificate from in our [metadata endpoint](#metadata) and below for verification.
 
 ### Metadata
 
-Consistent with the [SAML metadata specification](https://docs.oasis-open.org/security/saml/v2.0/saml-metadata-2.0-os.pdf), login.gov's metadata is available at [https://idp.int.identitysandbox.gov/api/saml/metadata2020](https://idp.int.identitysandbox.gov/api/saml/metadata2020)
+Consistent with the [SAML metadata specification](https://docs.oasis-open.org/security/saml/v2.0/saml-metadata-2.0-os.pdf), login.gov's metadata for our **sandbox** environment is available at [https://idp.int.identitysandbox.gov/api/saml/metadata2020](https://idp.int.identitysandbox.gov/api/saml/metadata2020).
 
-### Annual Certificate Rotation
+### Signing Certificates
```

Comment:
Let's add a note that this is the IDP certificate, not the SP certificate. To create the SP certificate "Go here" link to testing tab

Reply:
Done

Reply:
So it turns out this didn't render correctly.

And then I realized we already have accordions on this page ("View example authentication request" etc) can we just use those instead?

Reply:
@zachmargolis Ha saved by the bell, I was about to merge. I don't have strong preferences but I think I got the details/summary thing to work - this will be more concise I think. Any concerns with just merging as-is? ([preview here](https://federalist-14ccf384-8bd5-4e30-aa95-a5fe5894e3c1.app.cloud.gov/preview/18f/identity-dev-docs/oyk-saml-cert/saml/#signing-certificates))

Reply:
Preview looks good, merging as-is would be fine, but I think we should follow-up to unify the page and make it consistent with itself, so if that's accordion or details/summary I don't feel strongly

Reply:
Eh I'll just switch it - more likely to happen now than later 😄